client/package.json

```diff
@@ -94,6 +94,7 @@
     "build:dev": "NODE_ENV=development webpack --config webpack.config.coffee",
     "build:prod": "NODE_ENV=production webpack --config webpack.config.coffee --bail",
     "watch:dev": "NODE_ENV=development webpack --config webpack.config.coffee --colors --debug --watch",
+    "watch:prod": "NODE_ENV=production webpack --config webpack.config.coffee --colors --debug --watch",
```

Comment:
Nothing special, just to have a way to try changes with production build without having to rebuild from scratch every time.